### DISCUSSION TITLE
Error accessing WebGUI - Container is running on UnRaid without errors

### DISCUSSION THREAD

**Discussion by AbhiOommen** [linuxserver/Heimdall#1282] [App issues]
Hello,

I'm getting this when trying to access the GUI - 
![image](https://github.com/linuxserver/Heimdall/assets/11384002/7b029b03-27c8-479b-8ce9-22fc4874e9d5)

**Comment by locksen-nl** [comment]
Same

**Comment by KodeStar** [comment]
Should be fixed
